docs/upgrade-packages.md

```diff
@@ -0,0 +1,62 @@
+# Upgrading packages
+
+Updgrading packages should be as easy as possible. We assume that our test suite will bring up all the problems that may arise
+
+## Default process, the happy path
+
+```shell
+$ docker-compose run --rm react yarn upgrade --latest
+$ git commit package.json yarn.lock -m "Upgrade packages: yarn upgrade --latest"
+```
+
+Then just check that the tests pass. And that's it!
+
+**NOTE:** After this kind of update everybody should rebuild their docker images (`docker-compose build`) and/or run `yarn install` on their local machine.
+
+## Security fixes
+
+Sometimes `yarn upgrade --latest` doesn't fix all the security issues.
+
+Then it's ok to use add the patched version of the package in the [resolutions in package.json](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/):
+
+```js
+"resolutions": {
```

Comment:
So these packages are something else than the latest?

Reply:
Yes. These are dependencies of our project dependencies. `yarn upgrade --latest` upgrades all *our* dependencies to the latest but not all the sub-dependencies to the latest. When e.g. `react-scripts` is updated then we don't need to list `acorn` in here anymore because it will be resolved also without this kind of "forcing".

This was just something I "read from the internet" and understood that is the thing to do in this kind of situations 😊 (see link above^ https://classic.yarnpkg.com/en/docs/selective-version-resolutions)